src/BlazorUI/Bit.BlazorUI/Scripts/Utils.ts

```
@@ -3,12 +3,16 @@
         public static MIN_MOBILE_WIDTH = 320;
         public static MAX_MOBILE_WIDTH = 600;
 
-        public static throttle(mainFunction: Function, delay: number) {
+        public static getBodyWidth() {
+            return document.body.offsetWidth;
+        }
+
+        public static throttle(fn: Function, delay: number) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)